src/xenia/gpu/gpu_flags.cc

```diff
@@ -20,6 +20,9 @@ DEFINE_path(
 
 DEFINE_bool(vsync, true, "Enable VSYNC.", "GPU");
 
+DEFINE_uint64(vsync_interval, 16,
```

Comment:
I fully agree with @illusion0001, I think we shouldn't be exposing an explicitly incorrect value, it should be either a floating-point 1000/60 (or 1/60), or, similar to how DXGI exposes the monitor's refresh rate, a numerator and a denominator (either for the refresh rate, in which case they would be 60 and 1, or for the interval, 1 and 60 — though this is merely a naming question).

Our vsync logic is mathematically broken anyway. We count the time _since the last wakeup_ that has resulted in a vsync signal. So, if sleep resolution is, for instance, 3ms, we may be triggering vsyncs at 18ms, 36ms, and so on. If the sleep timing resolution is 15ms, vsyncs will be at 30ms, 60ms. With 16ms resolution — roughly the resolution of the sleep timer on Windows without `timeBeginPeriod` — the actual interval will be pretty much random, sometimes 16ms and 32ms.

Instead, at every wakeup, we need to trigger `(current time - last vsync time) / vsync interval` signals, and add `signal count * vsync interval` to the last vsync time (but maybe limit the number of actual signals sent to the game to 1 in case the user specifies some extremely low value, or if it's significantly lowered mid-game, by loading the game config — though Windows's ~16ms timer resolution without `timeBeginPeriod` may result in 2 signals for a wakeup, for instance, if wakeups happen at 15ms and at 15ms+18ms, the limit should be 2 in this case probably). This will make sure that if a wakeup happened, for instance, 18ms after the last vsync, the next will happen after at least 14ms, not 16ms. Note that this must be done with fixed-point values (integers — microseconds or nanoseconds), not with floating-point values, first, because absolute time is involved (so precision doesn't drop over time), and also so after we add `signal count * vsync interval` to the last vsync time, dividing the result by the vsync interval always gives 0, not sometimes 1 due to rounding.

Since time overall needs to be calculated as fixed-point, that's why it would be highly preferable to expose the interval as a rational number, not as a number (arbitrarily) rounded to a 23e8/53e11 float.

Also, in scenarios where the system's sleep resolution is too low (UWP on Windows, for instance, without `timeBeginPeriod`), we may need to fall back to some other kind of waiting (a yielding busy loop) — @JoelLinn should know a lot about this, I think.